src/component/Navbar.js

```diff
@@ -1,7 +1,80 @@
 import React from "react";
+import "./NavBar.css";
 
-const Navbar = () => {
-  return <div>Navbar</div>;
+export default function Navbar () {
+  return (
+  <>
+      <div>
+        <nav
+          className="navbar navbar-expand-lg" >
+          <a className="navbar-brand" href="#">
+            <img className="logo" src="/images/logo.jpg" alt="Logo" />
```

Comment:
From figma save the logo as png image, it will be much easy to work with and match it with the design